pkgs/test/README.md

```diff
@@ -1,3 +1,5 @@
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/dart-lang/test/badge)](https://api.securityscorecards.dev/projects/github.com/dart-lang/test)
```

Comment:
Is there any more visual page we can link to instead of the JSON scorecard result?

Reply:
Ok cool, as it stands its very confusing to understand what the badge represents at all.